modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/filters/Filters.tsx

```diff
@@ -133,20 +140,24 @@ function AddFDSFilterModalContent({
 
 		onSave({...responseJSON, displayType, filterType});
 
-		closeModal();
+		dismiss();
```

Comment:
Do we need `dismiss` now? Can the setting of active mode now we just a part of `onSave`?

Reply:
Do you mean calling `setActiveMode(FILTER_MODE.LIST)` here instead?

Reply:
Isn't `onSave` more natural, as `activeMode` is state in parent scope?

Both `dismiss` and `onSave` are callbacks on `FilterFormComponent`, executed here one after another. Is there a reason not to merge them?